src/amuse/datamodel/base.py

```diff
@@ -616,15 +616,15 @@ def __setstate__(self, data):
         
     def __str__(self):
         lines = []
-        for name, value in self._attributes.iteritems():
+        for name, value in self._attributes.items():
             lines.append("{0}: {1}".format(name, value))
         return '\n'.join(lines)
     
     def _copy_for_collection(self, newcollection):
         return CollectionAttributes(self._attributes)
         
     def iteritems(self):
-        return self._attributes.iteritems()
+        return iter(self._attributes.items())
```

Comment:
this one I dont understand...

Reply:
also it looks like the name of the method is dangerous..on the otherhand, if this goes wrong it will generate an easy debugged error..

Reply:
Related to #489 in that it is caused by 2to3 possibly being too careful